homeassistant/components/flux_led/light.py

```diff
@@ -333,6 +331,13 @@ def turn_on(self, **kwargs):
         if brightness is None:
             brightness = self.brightness
 
+        # If these are 0 then bulb.isOn will return false, not what we want
+        if hs_color is None or (hs_color[0] == 0 and hs_color[1] == 0):
+            hs_color = (1, 1, 1)
```

Comment:
`hs_color` from kwargs is a tuple with two elements (hue + saturation), where did the third element come from?
also, `hs_color` is not used after this line.

Maybe the intention is to instead check if all elements of rgb are 0, if inf they are, set rgb to `(1, 1, 1)` or `(255, 255, 255)`?